templates/zerver/integrations/git.md

```diff
@@ -8,24 +12,23 @@ Get Zulip notifications for your Git repositories!
 
 1. {!change-zulip-config-file.md!}
 
-    You may also need to change the value of `STREAM_NAME`.
+    Also update the value of `STREAM_NAME` to the name of your channel.
```

Comment:
Ideally this would just be part of the block above, rather than a separate instruction. So you'd have something like:

> STREAM_NAME = "<Name of the channel where messages should be sent"